X/xxHash_jll/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "xxHash_jll"
+version = v"0.7.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Cyan4973/xxHash.git", "173e50be0509c6fb6c1bb74d95049ef61d7fdced")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd xxHash/
+CPPFLAGS=-DXXH_INLINE_ALL make
+if [ ! -d $libdir ]; then
+  mkdir -p $libdir;
```

Comment:
For the record, the `-p` flag to `mkdir` causes it to not throw an error if the directory already exists, so there is no need to put it under a conditional like this